### DISCUSSION TITLE
still getting blank images if using a image prompt

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#3445] [Q&A]
![2024-08-03_12-54-33_8592](https://github.com/user-attachments/assets/553e2e5e-afa3-4974-b2e5-5372bbaac9fb)
```
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
	- Avoid using `tokenizers` before the fork if possible
	- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
/Users/Quad-Core/Fooocus/modules/anisotropic.py:132: UserWarning: The operator 'at![2024-08-03_12-54-33_8592](https://github.com/user-attachments/assets/553e2e5e-afa3-4974-b2e5-5372bbaac9fb)ormance implications. (Triggered internally at /Users/runner/work/_temp/anaconda/conda-bld/pytorch_1711403210267/work/aten/src/ATen/mps/MPSFallback.mm:13.)
  s, m = torch.std_mean(g, dim=(1, 2, 3), keepdim=True)
/Users/Quad-Core/Fooocus/modules/core.py:257: RuntimeWarning: invalid value encountered in cast
  x_sample = x_sample.cpu().numpy().clip(0, 255).astype(np.uint8)
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 60/60 [15:28<00:00, 15.48s/it]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.48 seconds
/Users/Quad-Core/Fooocus/modules/core.py:331: RuntimeWarning: invalid value encountered in cast
  return [np.clip(255. * y.cpu().numpy(), 0, 255).astype(np.uint8) for y in x]
[Fooocus] Saving image 1/1 to system ...
Image generated with private log at: /Volumes/1TB Caddy bay 5/2024-08-04/log.html
Generating and saving time: 942.79 seconds
Total time: 975.12 seconds
```

image is blank, had this for months now and out of no where, used to use a image all the time.

Prompt	A English rose woman, blonde hair, blue eyes, freckles around nose and cheeks. 32dd breasts, perfect eyes, perfect hands sitting in a tattoo chair
Negative Prompt	asymmetry
Fooocus V2 Expansion	A English rose woman, blonde hair, blue eyes, freckles around nose and cheeks. 32dd breasts, perfect eyes, perfect hands sitting in a tattoo chair, sharp focus, intricate, elegant, highly detailed, cute, innocent, sublime, dramatic light, professional, pleasant, bright color, inspired, pretty, attractive, best,, smart, charming, creative
Styles	['Fooocus V2', 'Fooocus Enhance', 'Fooocus Sharp']
Performance	Quality
Resolution	(1152, 896)
Guidance Scale	4
Sharpness	2
ADM Guidance	(1.5, 0.8, 0.3)
Base Model	juggernautXL_v8Rundiffusion.safetensors
Refiner Model	None
Refiner Switch	0.5
CLIP Skip	2
Sampler	dpmpp_2m_sde_gpu
Scheduler	karras
VAE	Default (model)
Seed	1593615615078291072
LoRA 1	JessicaL.safetensors : 1.0
LoRA 2	PerfectEyesXL.safetensors : 1.0
LoRA 3	Perfect Hands v2.safetensors : 1.0
Metadata Scheme	False
Version	Fooocus v2.4.3
Full raw prompt	
Positive
cinematic still A English rose woman, blonde hair, blue eyes, freckles around nose and cheeks. 32dd breasts, perfect eyes, perfect hands sitting in a tattoo chair . emotional, harmonious, vignette, 4k epic detailed, shot on kodak, 35mm photo, sharp focus, high budget, cinemascope, moody, epic, gorgeous, film grain, grainy, A English rose woman, blonde hair, blue eyes, freckles around nose and cheeks. 32dd breasts, perfect eyes, perfect hands sitting in a tattoo chair, sharp focus, intricate, elegant, highly detailed, cute, innocent, sublime, dramatic light, professional, pleasant, bright color, inspired, pretty, attractive, best,, smart, charming, creative
Negative

**Comment by mashb1t** [comment]
As already mentioned, might have to do with an update of your MacOS or updated drivers, but it works for me on 14.5/14.6 on a Macbook Pro M1 without any issues.

**Reply by mashb1t** [reply]
Related to https://github.com/lllyasviel/Fooocus/discussions/3069 / https://github.com/lllyasviel/Fooocus/issues/1184 / https://github.com/lllyasviel/Fooocus/pull/129
